styles/styles.css

```diff
@@ -249,6 +260,18 @@ main .section .default-content-wrapper h4 {
   animation: spinner-spin 2s linear infinite;
 }
 
+.btn.btn-secondary {
```

Comment:
These aren't classes that an author can add to any configuration. I've not seen them anywhere but in the context of the search, so i'm not sure if they need to be global?

Reply:
removed